backend/lib/endpoints/feedback.js

```diff
@@ -0,0 +1,44 @@
+const { createFeedbackSchema } = require("./schema/feedback");
+
+/*
+ * /api/feedback
+ */
+async function routes(app) {
+  const Feedback = app.mongo.model("Feedback");
+
+  app.post("/", { schema: createFeedbackSchema }, async (req, reply) => {
+    const { userId } = req.body;
+    let ip = "undefined";
+    if (userId) {
+      const userFeedback = await Feedback.find({ userId });
+      if (userFeedback[0]) {
+        reply.conflict("User already submitted feedback");
+      }
+    }
+
+    if (req.ip !== null && typeof req.ip !== "undefined") {
+      ip = req.ip;
+      const feedbackFromIp = await Feedback.find({ ipAddress: ip });
+      if (feedbackFromIp[0]) {
+        reply.conflict("Feedback already submitted with this IP address");
+      }
+    }
+
+    if (userId || ip !== "undefined") {
+      return new Feedback({
```

Comment:
for consistency probably also better to stick with await syntax here, but we do have to find a good way to use the `reply.code()` combined with the `model.save()` 
I think we should try to use this  "to" utility from fastify-sensible since it's exactly what we need and then we can just `reply.code();` and `return ...` on success.
https://github.com/fastify/fastify-sensible#to